### PR TITLE
Fix mobile view background color

### PR DESCRIPTION
**Pull request by cychiuae**
ref DEV-1464

Screenshot 2024-06-28 at 11 12 57 PM
Screenshot 2024-06-28 at 11 13 09 PM

### CODE REVIEW

**Comment by cychiuae** [comment]
@tung2744

**Comment by cychiuae** [comment]
Seems e2e test timed out and panic. I think it is not related to the code change i made 🤔

File: authui/src/authflowv2/components/widget.css [hunk 1]
```diff
@@ -16,18 +16,30 @@
...
   :root:not([alignment-content]) .widget {
     /* This rule is to prevent ui shifting before the attribute added */
     @apply invisible;
   }
+
+  @media (min-width: theme("screens.tablet")) {
+    .widget {
+      max-width: var(--widget__max-width);
+    }
+  }
```
[the remark this answers is not in the document]
Comment (tung2744) [reply authui/src/authflowv2/components/widget.css:44]: I think the approach in this pr is better.

**Comment by tung2744** [comment]
This is a known issue that e2e tests will sometimes stuck.
I triggered it again and it successfully passed.

**Review by tung2744 — APPROVED** [review]

**Pull request merged**